Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35428&attempt_id=8306af1f-12c5-42e8-806e-a24b33dbec81)

<details><summary>Partial log (click to expand)</summary><p>

```
adplug/fmopl.c:599:2: note: within this loop
  for (i = 60;i < 76;i++)
  ^~~
make[2]: Leaving directory '/build/deadbeef-0.7.2/plugins/adplug'
make[1]: *** [Makefile:769: all-recursive] Error 1
make[1]: Leaving directory '/build/deadbeef-0.7.2'
make: *** [Makefile:527: all] Error 2
builder for '/nix/store/j98h0ny3yfr908ddzx3ppw1wr4qg32jn-deadbeef-0.7.2.drv' failed with exit code 2
cannot build derivation '/nix/store/wf0gdza28ip1r39n8g8ww619xabdsihh-deadbeef-opus-plugin-0.8.drv': 1 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/wf0gdza28ip1r39n8g8ww619xabdsihh-deadbeef-opus-plugin-0.8.drv' failed
```
</p></details>

--- a/pkgs/applications/audio/deadbeef/plugins/opus.nix
+++ b/pkgs/applications/audio/deadbeef/plugins/opus.nix
@@ -1,0 +1,31 @@
+{ stdenv, fetchFromBitbucket, opusfile, libopus, libogg, openssl, deadbeef }:
+
+stdenv.mkDerivation rec {
+  name = "deadbeef-opus-plugin-${version}";
+  version = "0.8";
+
+  src = fetchFromBitbucket {
+    owner = "Lithopsian";
+    repo = "deadbeef-opus";
+    rev = "v${version}";
+    sha256 = "057rgsw4563gs63k05s7zsdc0n4djxwlbyqabf7c88f23z35ryyi";
+  };
+
+  makeFlags = [
+    "PREFIX=$(out)"
+  ];
+
+  NIX_CFLAGS_COMPILE = [
+    "-I${opusfile}/include/opus"
+  ];
+
+  buildInputs = [ deadbeef opusfile libopus libogg openssl ];
+
+  meta = with stdenv.lib; {
+    description = "Ogg Opus decoder plugin for the DeaDBeeF music player";
+    homepage = https://bitbucket.org/Lithopsian/deadbeef-opus;
+    license = licenses.gpl2; # There are three files, each licensed under different license: zlib, gpl2Plus and lgpl2
+    maintainers = [ maintainers.jtojnar ];
+    platforms = platforms.linux;
+  };
+}

--- a/pkgs/top-level/aliases.nix
+++ b/pkgs/top-level/aliases.nix
@@ -49,6 +49,7 @@ mapAliases (rec {
   cupsBjnp = cups-bjnp; # added 2016-01-02
   cups_filters = cups-filters; # added 2016-08
   cv = progress; # added 2015-09-06
+  deadbeef-mpris2-plugin = deadbeefPlugins.mpris2; # added 2018-02-23
   debian_devscripts = debian-devscripts; # added 2016-03-23
   digikam5 = digikam; # added 2017-02-18
   double_conversion = double-conversion; # 2017-11-22

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -14872,7 +14872,10 @@ with pkgs;
     pulseSupport = config.pulseaudio or true;
   };
 
-  deadbeef-mpris2-plugin = callPackage ../applications/audio/deadbeef/plugins/mpris2.nix { };
+  deadbeefPlugins = {
+    mpris2 = callPackage ../applications/audio/deadbeef/plugins/mpris2.nix { };
+    opus = callPackage ../applications/audio/deadbeef/plugins/opus.nix { };
+  };
 
   deadbeef-with-plugins = callPackage ../applications/audio/deadbeef/wrapper.nix {
     plugins = [];

